From a33d84566e90cc279db7cc54de45ab92ce10bdf6 Mon Sep 17 00:00:00 2001
From: Andrey Konovalov <andrey.konovalov@linaro.org>
Date: Wed, 15 Oct 2014 14:49:53 +0400
Subject: [PATCH 25/25] ARM64:ILP32: fix the compilation error due to mistyped
 header file name

There is a misprint in the v3 patch "ARM64:ILP32: Use the same size and
layout of the signal structures for ILP32 as for LP64".

Signed-off-by: Andrey Konovalov <andrey.konovalov@linaro.org>
---
 arch/arm64/include/uapi/asm/signal.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/include/uapi/asm/signal.h b/arch/arm64/include/uapi/asm/signal.h
index d33a5f7..f378ff4 100644
--- a/arch/arm64/include/uapi/asm/signal.h
+++ b/arch/arm64/include/uapi/asm/signal.h
@@ -16,7 +16,7 @@
 #ifndef __ASM_SIGNAL_H
 #define __ASM_SIGNAL_H
 
-#include <asm/posix-types.h>
+#include <asm/posix_types.h>
 
 /* Required for AArch32 compatibility. */
 #define SA_RESTORER	0x04000000
-- 
1.9.3

